tests/core/test_agent.py

```diff
@@ -385,6 +390,7 @@ async def test_agent_update_model_none_domain(trained_rasa_model: Text):
     assert tracker.events[3].intent["name"] == "greet"
 
 
+@pytest.mark.trains_model
```

Comment:
why is this one marked?

Reply:
oops was meant to be on `test_load_agent`